packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -505,6 +520,35 @@ export default {
             }
         },
 
+        /**
+         * Look up the geo details for the customers address, skip on failure.
+         *
+         */
+        async lookupGeoLocation () {
+            const postData = { addressLines: [] };
+            Object.keys(this.address).forEach(propName => {
```

Comment:
I'd rename `propName` to something else, given that `prop` in Vue's context refers normally to `props` and not `property` :)